app/js/actions/createLabOrder.js

```diff
@@ -0,0 +1,41 @@
+import 'babel-polyfill';
+import randomString, { generate } from 'randomstring';
+
+import loading from './loading';
+import axiosInstance from '../config';
+
+import {
+  CREATE_LAB_ORDER,
+  CREATE_LAB_ORDER_SUCCESS,
+  CREATE_LAB_ORDER_FAILURE,
+} from './actionTypes';
+
+const createLabOrderFailure = error => ({
+  type: CREATE_LAB_ORDER_FAILURE,
+  error,
+});
+
+const createLabOrderSuccess = data => ({
+  type: CREATE_LAB_ORDER_SUCCESS,
+  data,
+});
+
+const createLabOrder = labOrderData => async (dispatch) => {
+  try {
+    dispatch(loading(CREATE_LAB_ORDER, true));
+    const requestPayload = {
+      name: randomString.generate(10),
+      description: JSON.stringify(labOrderData),
```

Comment:
I see you are using a random string as the name of the lab order, I am not sure this is supposed to be.
If it is supposed to be, please provide a reason for that.
If it is not supposed to be, I believe you should be able to get the name of the Lab Order from the details provided by the user.

Reply:
To save into the order type resource, the name field has to be unique, so all the data relating to the lab order is in the description field and the random string simply serves as a placeholder for a name in the database,
again, this is just an alternative until the bug with the order resource endpoint is fixed.

Reply:
If the reason you are using a random string is the bug you are facing with storing data through the `order` endpoint, please follow up with the discussion in this post on how to go about fixing the bug:  https://talk.openmrs.org/t/issue-making-a-post-request-to-create-a-test-order-order-type/18908/7?u=tunmi